Tests/Marketplace/search_and_install_packs.py

```diff
@@ -634,13 +645,16 @@ def search_and_install_packs_and_their_dependencies_private(test_pack_path: str,
 
 
 def search_and_install_packs_and_their_dependencies(pack_ids: list,
-                                                    client: demisto_client, hostname: str = ''):
+                                                    client: demisto_client, hostname: str = '',
+                                                    install_packs_one_by_one=False):
```

Comment:
Same suggestion as above - Change to an `int` parameter of the batch_size.

Reply:
I did it before, but decided it is not so good practice, https://github.com/demisto/content/pull/23902/commits/d76e115ad2d8c8ce8bdd224007aa22135ab29381
 
 I can explain why on zoom call :)